core/tabs/applications-setup/mybash-setup.sh

```diff
@@ -5,16 +5,16 @@
 gitpath="$HOME/.local/share/mybash"
 
 installDepend() {
-    if ! command_exists bash bash-completion tar bat tree unzip fontconfig git; then
-    printf "%b\n" "${YELLOW}Installing Bash...${RC}"
-    case "$PACKAGER" in
-        pacman)
-            "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm bash bash-completion tar bat tree unzip fontconfig git
-            ;;
-        *)
-            "$ESCALATION_TOOL" "$PACKAGER" install -y bash bash-completion tar bat tree unzip fontconfig git
-            ;;
-    esac
+    if [ ! -f "/usr/share/bash-completion/bash_completion" ] || ! command_exists bash tar bat tree unzip fc-list git; then
+        printf "%b\n" "${YELLOW}Installing Bash...${RC}"
+        case "$PACKAGER" in
+            pacman)
+                "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm bash bash-completion tar bat tree unzip fontconfig git
+                ;;
+            *)
+                "$ESCALATION_TOOL" "$PACKAGER" install -y bash bash-completion tar bat tree unzip fontconfig git
+                ;;
+        esac
```

Comment:
```suggestion
    if ! command_exists bash tar bat tree unzip git; then
    printf "%b\n" "${YELLOW}Installing Bash...${RC}"
    case "$PACKAGER" in
        pacman)
            "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm bash bash-completion tar bat tree unzip fontconfig git
            ;;
        *)
            "$ESCALATION_TOOL" "$PACKAGER" install -y bash bash-completion tar bat tree unzip fontconfig git
            ;;
    esac
```

just remove them from command_exists

Reply:
we shouldnt add unnecessary overhead, this is a much simpler way

Reply:
Removing them is incorrect. What if the user already has every other deps installed but not bash-completion ?

Reply:
Checking if a file exists does not create extra overhead 🤷🏻‍♂️

Reply:
mark this as resolved